compiler/rustc_ast/src/token.rs

```diff
@@ -452,6 +452,8 @@ impl Token {
             || self.is_keyword(kw::Enum)
             || self.is_keyword(kw::Struct)
             || self.is_keyword(kw::Union)
+            || self.is_keyword(kw::Macro)
+            || self.is_keyword(kw::MacroRules)
```

Comment:
MacroRules shouldn't be here.

Reply:
[Just noticed, this probably shouldn't be a "Suggested changes", haha.]

Reply:
yes, should remove MacroRules.